lib/WUI/nhttp/status_page.cpp

```diff
@@ -20,10 +21,19 @@ Step StatusPage::step(std::string_view, bool, uint8_t *output, size_t output_siz
     const StatusText &text = StatusText::find(status);
 
     char content_buffer[128];
+    ContentType ct;
```

Comment:
```suggestion
    ContentType ct = ContentType::TextPlain;
```

Reply:
I can understand why you don't want an unitialized variable around, so I'm giving it some initial value now (corresponding to "unknown"). But I believing having one branch of the if special and defaulting the value there by not setting it hurts readability. See the current version.